### ISSUE TITLE
Fix indentation

### ISSUE THREAD

**Issue by paulrouget** [mozilla/browser.html#89]
Indentation is a mess. I'm not sure exactly what rules we're following in the JS files.

For example:

```javascript
define((require, exports, module) => {

'use strict'; // Why don't we start indenting here?
[…]
const NavigationPanel = Component(({key, input, tabStrip, webViewer, title}) => {
  return DOM.div({
    key,
    className: 'navbar' + (input.get('isFocused') ? ' urledit' : '')
[…]
  }, [
    WindowControls({key: 'controls'})
[…]
  ])
  }); // Why is this indented? 
exports.NavigationPanel = NavigationPanel;

});
```

Can we define some rules? Maybe just use what we're used to with Firefox source code?

/cc @Gozala

**Comment by gordonbrander** [comment]
Isn't this closed by #101?

**Comment by paulrouget** [comment]
Yes.
